include/librealsense2/h/rs_sensor.h

```diff
@@ -90,6 +90,7 @@ typedef enum rs2_format
     RS2_FORMAT_FG              , /**< 16-bit per-pixel frame grabber format. */
     RS2_FORMAT_Y411            , /**< 12-bit per-pixel. */
     RS2_FORMAT_Y16I            , /**< 12-bit per pixel interleaved. 12-bit left, 12-bit right. */
+    RS2_FORMAT_NV12I           , /**< 24-bit y for every pixel, and u, v data for every four pixels. Similar to YUV420 but packed in a different order */
```

Comment:
Like we already know it's not similar to YUV420
Lets say:
A diverged custom version to native NV12.
And elaborate the actual changes in the unpacker.
What do you think?

Reply:
Done